### DISCUSSION TITLE
前期挂载好的115，这两天报错了，视频不能播放。

### DISCUSSION THREAD

**Discussion by majiancy** [AlistGo/alist#6895] [Q&A] [answered]
发现了如题问题，就登到alist后台看了一下，提示是因为其他终端登录了balabala...，以前遇到这个问题就重新ScanQR拿到一个新的token填入连接即可。但这次如上操作后，依然报错（信息如下），是115下线了什么？原来的方法不能用了吗？

`Failed init storage: failed to login by qrcode: {"state":0,"error":"登录失败，系统已下架！如果你有电脑端的使用需求，我们诚挚邀请你下载体验115产品专属客户端“115浏览器”或在线使用“115网页端(115.com)”，畅享智能高效云生活。","errno":0,"message":"登录失败，系统已下架！如果你有电脑端的使用需求，我们诚挚邀请你下载体验115产品专属客户端“115浏览器”或在线使用“115网页端(115.com)”，畅享智能高效云生活。","code":0}: unexpected error`

**Comment by dreamoeu** [comment, marked as answer]
![Screenshot_2024-07-29-00-03-56-859_com.ylmf.androidclient-edit.jpg](https://github.com/user-attachments/assets/01964efe-2552-463a-95f0-d5f8a4d4f48d)

**Comment by majiancy** [comment]
因此，目前没有其他办法呗，难道都要迁移到pikpak去？

**Reply by solo-0x** [reply]
抓包手机端115的Cookie

**Reply by san346596324** [reply]
> 抓包手机端115的Cookie

有什么好用的安卓端抓包工具吗
